dashboard/src/pages/TicketTransfer.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+  <Header />
+  <div class="w-full flex items-center justify-center">
+    <TransferSuccess v-if="inSuccess" />
+    <div v-else class="max-w-screen-xl w-full p-5">
+      <div class="mt-8">
+        <div class="prose">
+          <div class="text-base uppercase font-medium text-gray-600 mb-2">
+            Portal
+          </div>
+          <h1 class="m-0">Ticket Transfers</h1>
+          <div
+            v-if="transfer_settings.doc"
+            v-html="transfer_settings.doc.web_page_description"
+          ></div>
+        </div>
+      </div>
+      <div class="bg-white p-6 rounded border w-full">
+        <div class="flex flex-col md:grid md:grid-cols-2 gap-4">
+          <div class="col-span-2 flex flex-col gap-2">
+            <div class="prose">
+              <h4>Ticket Details</h4>
+            </div>
+            <div class="w-full flex flex-col md:grid md:grid-cols-2 gap-4">
+              <FormControl
+                size="sm"
+                label="Ticket ID  &ast;"
+                v-model="ticketId"
+                type="text"
+                @focusout="
+                  () => {
+                    isTicketValid()
+                  }
+                "
+              />
+              <div class="hidden md:block"></div>
+              <div
+                v-if="ticket.data && event.data"
+                class="col-span-2 grid grid-cols-1 md:grid-cols-2 gap-4"
+              >
+                <FormControl
+                  label="Event"
+                  v-model="event.data.event_name"
+                  type="text"
+                  :disabled="true"
+                />
+                <FormControl
+                  label="Ticket Tier"
+                  v-model="ticket.data.tier"
+                  type="text"
+                  :disabled="true"
+                />
+              </div>
+            </div>
+            <ErrorMessage :message="ticketValidateError" />
+          </div>
+          <div class="col-span-2">
+            <hr />
+          </div>
+          <div class="col-span-2 prose">
+            <h4>Reciever Details</h4>
+          </div>
+          <FormControl
+            v-model="reciever.reciever_email"
+            label="Reciever Email &ast;"
+            type="email"
+            size="sm"
+            placeholder="john@fossunited.org"
+          />
+          <FormControl
+            v-model="reciever.reciever_name"
+            label="Reciever Name &ast;"
```

Comment:
Typo in this - Reciever -> Receiver 

Needs to be changed in other places too.

Reply:
Outdated commit. Changed in later ones

Reply:
Okay - please update the screenshots in PR description too - it shows the typo